saleor/graphql/menu/mutations.py

```diff
@@ -196,6 +198,103 @@ def user_is_allowed(cls, user, input):
         return user.has_perm('menu.manage_menus')
 
 
+_MenuMoveOperation = namedtuple(
+    '_MenuMoveOperation', ('menu_item', 'parent', 'sort_order'))
+
+
+class MenuItemMove(BaseMutation):
+    menu_item = graphene.List(
+        MenuItem, description='Assigned menu to move within.')
+
+    class Arguments:
+        moves = graphene.List(
+            MenuItemMoveInput,
+            required=True, description='The menu position data')
+
+    class Meta:
+        description = 'Moves items of menus'
+
+    @staticmethod
+    def pre_validate_move(move):
```

Comment:
To not introduce new wording I'd stick with good ol' "clean" and have here `clean_move`. It's easier to read.